config/csv/config.go

```diff
@@ -33,6 +33,20 @@ func EmptyNull(emptyNull bool) ConfigFunc {
 	}
 }
 
+// MissingColumnNameAlias sets the name to be used for empty columns name with given string
+func MissingColumnNameAlias(MissingColumnNameAlias string) ConfigFunc {
+	return func(c *Config) {
+		c.MissingColumnNameAlias = MissingColumnNameAlias
+	}
+}
+
+// AddColumnIndexToDuplicateColumns configures if duplicate column names should have the column index appended to the column name to resolve the conflict.
+func AddColumnIndexToDuplicateColumns(AddColumnIndexToDuplicateColumns bool) ConfigFunc {
```

Comment:
As mentioned elsewhere please rename this configuration option now that the column indices are not part of the solution. `RenameDuplicateColumns`?